pages/price-feeds/sponsored-feeds.mdx

```diff
@@ -0,0 +1,26 @@
+# Sponsored Feeds
+
+The Pyth Data Association sponsors price updates for several price feeds in Solana mainnet and devnet.
```

Comment:
Can we highlight Solana here for now? Or maybe we can add this into chain specific page/section?